docs: document broken upgrade version for tasklist (#5858)
camunda/camunda-docs: docs/self-managed/operational-guides/update-guide/840-to-850.md

Identity now uses the Keycloak frontend URL instead of the backend URL. This change may affect you if you have blocked the Keycloak frontend URL from other services (including Camunda applications), and can potentially impact Identity's functionality.

To avoid connectivity issues, ensure your Keycloak frontend URL is accessible by adjusting your network, firewall, or security settings as needed. This adjustment is crucial to maintain the integration with Keycloak and ensure compatibility.

## Tasklist

:::caution

> There is a [bug](https://github.com/camunda/camunda/issues/32208) in Tasklist 8.5.15 which causes Tasklist's importer to fail importing records for the `flownode-instance` index. As such, **you should skip 8.5.15 and update directly to 8.5.16**

Reviewer: ```suggestion
**You should skip 8.5.15 and update directly to 8.5.16** due to a [bug](https://github.com/camunda/camunda/issues/32208) in Tasklist 8.5.15 that causes Tasklist's importer to fail importing records for the `flownode-instance` index.
```
